app/scripts/modules/kubernetes/src/v2/manifest/selector/ManifestSelector.tsx

```diff
@@ -179,7 +179,8 @@ export class ManifestSelector extends React.Component<IManifestSelectorProps, IM
       .map(([kind]) => kind)
       .sort();
 
-    if (namespaces.every(ns => ns !== this.state.selector.location)) {
+    const isLocationExpression = this.state.selector.location.includes('${');
```

Comment:
Could use `this.isExpression(...)` instead

Reply:
Oh duh didn't notice that method, thanks Dan!